ovn-tester/ovn_tester.py

```diff
@@ -236,6 +250,23 @@ def read_config(config):
     return global_cfg, cluster_cfg, brex_cfg, bringup_cfg
 
 
+def setup_logging(global_cfg):
+    FORMAT = '%(asctime)s | %(name)-12s |%(levelname)s| %(message)s'
+    logging.basicConfig(stream=sys.stdout, level=logging.INFO, format=FORMAT)
```

Comment:
Are you referring to the `__main__` on that first line? That's because we get our logger using `logging.getLogger(__name__)`. In imported modules, `__name__` is the name of the module. In the entry point python file, `__name__` evaluates to `__main__` . A fix for this would be to use the filename or the constant "ovn-tester" instead.